docroot/themes/humsci/humsci_basic/src/scss/utilities/_general.scss

```diff
@@ -279,3 +279,18 @@ details.hb-accordion-light {
 .hb-clear {
   clear: both;
 }
+
+// Fix tables in accordions
+.ptype-hs-accordion details {
+  &[open] {
+    .revealed-details {
+      .hb-table-wrap {
+        overflow-x: hidden;
+      }
+
+      .hb-table-pattern {
+        width: auto;
+      }
+    }
+  }
+}
```

Comment:
The `overflow-x: auto` is intentional, I'm pretty sure it's there to make the table readable in all resolutions. I'll check with Tori if that's something they want to keep, but for now let's remove this change to keep the overflow auto.

Also, `width: auto` on the table could make small tables to shrink.

There's a bug that needs to be fixed though. Right now all tables have a scrollbar, because they have a 100% width but the `box-sizing` is set to `content-box`. This makes the border not being included in the 100% and causing the table to always overflows by 2px.

To fix it, let change the `box-sizing` for the table element:

```css
.hb-table-pattern {
  // ...
  box-sizing: border-box;
}
```

Let's fix this globally in `docroot/themes/humsci/humsci_basic/src/scss/components/_pattern.table-pattern.scss`.